proto/viam/app/v1/app.proto

```diff
@@ -146,6 +146,17 @@ service AppService {
 
   // Deletes a fragment
   rpc DeleteFragment(DeleteFragmentRequest) returns (DeleteFragmentResponse);
+
+  // Billing
+
+  // Creates a Stripe customer
+  rpc CreateStripeCustomer(CreateStripeCustomerRequest) returns (CreateStripeCustomerResponse);
```

Comment:
Hm do we need an RPC for this? My assumption was that we can embed this within our current CreateOrganization grpc call

Reply:
yeah, unless there is a specific reason for this to exist as something the FE must call don't think we need this.